conf/messages.en

```diff
@@ -257,7 +258,7 @@ packAtBusinessAddress.title = Your business address
 packAtBusinessAddress.heading2 = Is your registered business address a UK packaging site that you operate to produce liable drinks?
 packAtBusinessAddress.checkYourAnswersLabel = Is your registered business address a UK packaging site that you operate to produce liable drinks?
 packAtBusinessAddress.error.required = Select yes if packAtBusinessAddress
-packAtBusinessAddress.change.hidden = PackAtBusinessAddress
+packAtBusinessAddress.change.hidden = Pack At Business Address
```

Comment:
Same as above here should be the UK packaging site that you operate to produce liable drinks.